maps/astrafare/warfare_items.dm

```diff
@@ -0,0 +1,613 @@
+/obj/item/clothing/head/helmet/hard_had
```

Comment:
Instead of copying and pasting this file, we should just include the original file in your new map.